### DISCUSSION TITLE
devops/test/performance/11.0.3/docs/topics/r_readwrite_ds_perf

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#85] [Q&A]
# devops/test/performance/11.0.3/docs/topics/r_readwrite_ds_perf

When a test is associated with a dataset, you can extend the test either by reading or writing the dataset values from the custom code.

https://help.hcl-software.com/devops/test/performance/11.0.3/docs/topics/r_readwrite_ds_perf.html?hl=getdatasetcontroller

**Comment by nbrahimi** [comment]
In this function control.writeRow, what's the meaning of last parameter. What's the impact of setting it to true. 
Is there any page in this knowledge center I can go to to get some more details about these functions under IDataSetController. Thanks

**Reply by Lekha-N** [reply]
The last parameter in the control.writeRow function is the overwrite option. If it is set to true, the function will replace any existing row at the specified index with the new values provided. If set to false, the function will not overwrite any existing values; instead, if the specified index already contains a row, it will remain unchanged, and the new row will be inserted after the existing row. There isn't a specific documentation that details the functions under IDataSetController. Thanks!
